.github/workflows/scanner-db-init-dump.yaml

```diff
@@ -0,0 +1,115 @@
+name: Scanner update database init dump
+
+on:
+  schedule:
+    # Run at midnight UTC.
+    - cron: "0 0 * * *"
+
+jobs:
+  build-updater:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:scanner-test-0.3.61
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Ignore dubious repository ownership
+        run: |
+          # Prevent fatal error "detected dubious ownership in repository" from recent git.
+          git config --global --add safe.directory "$(pwd)"
+
+      - name: Build updater
+        run: |
+          make tag
+          make -C scanner bin/updater
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: updater
+          path: scanner/bin/updater
+
+  build-init-dump:
+    needs:
+      - build-updater
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: registry.redhat.io/rhel8/postgresql-15
+        credentials:
+          username: ${{ secrets.RH_REGISTRY_USERNAME_RO }}
+          password: ${{ secrets.RH_REGISTRY_PASSWORD_RO }}
+        env:
+          POSTGRESQL_ADMIN_PASSWORD: scanner
+        # User "runner(1001)" owns the workspace directory.
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+          --user 1001
+        # Use workspace to ensure the volume permission is set to "runnner(1001)".
+        volumes:
+          - ${{ github.workspace }}:/var/lib/pgsql/data
+        ports:
+          - 5432:5432
+    env:
+      # TODO Hard-coded to "dev" while we don't have a matrix job to go over all
+      #      release branches.
+      version: dev
+    steps:
+      - uses: actions/download-artifact@v4
+        with:
+          name: updater
+          path: /usr/local/bin
+
+      - name: Install PostgreSQL 15 client
+        shell: bash
+        run: |
+          source /etc/os-release
+          echo "deb http://apt.postgresql.org/pub/repos/apt $VERSION_CODENAME-pgdg main" | sudo tee /etc/apt/sources.list.d/pgdg.list
+          curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+          sudo apt-get update
+          sudo apt-get install -y postgresql-client-15
+
+      - name: Run updater
+        run: |
+          chmod +x /usr/local/bin/updater
+          updater \
+              --vulns-url "https://storage.googleapis.com/scanner-v4-test/vulnerability-bundles/$version/vulns.json.zst" \
+              --db-conn 'host=localhost user=postgres database=postgres password=scanner'
```

Comment:
nit: we usually add the updater to an archive and untar it to preserve the permissions

Reply:
Yeah, I just thought that calling `chmod` here was less typing. :smiley: